app/src/main/java/org/schabi/newpipe/util/NavigationHelper.java

```diff
@@ -346,7 +334,7 @@ public static void openVideoDetailFragment(@NonNull final Context context,
 
         final boolean autoPlay;
         @Nullable final MainPlayer.PlayerType playerType = PlayerHolder.getInstance().getType();
-        if (playerType == null) {
+        if (PlayerHolder.getInstance().isPlayerOpen()) {
```

Comment:
You changed the meaning!
```suggestion
        if (!PlayerHolder.getInstance().isPlayerOpen()) {
```

Reply:
no good, sorry about that
hopefully obvious that it was not intentional.